demo/settings.yaml

```diff
@@ -5,9 +5,11 @@ output_path: './'
 
 # providers from which to search (case insensitive)
 providers:
-        #- 'GlassDoor' 
+
+  - 'GlassDoorDynamic'
   - 'Indeed'
-  - 'Monster'
```

Comment:
quick Q, but why are we changing this behaviour for Monster?

I think we disabled Monster and Glassdoor due to the web driver stuff? @thebigG

Reply:
Re-enabled Monster